data/model/test/test_tag.py

```diff
@@ -352,13 +353,22 @@ def test_get_most_recent_tag(initialized_db):
     # Create a hidden tag that is the most recent.
     repo = model.repository.get_repository("devtable", "simple")
     image = model.tag.get_tag_image("devtable", "simple", "latest")
-    model.tag.create_temporary_hidden_tag(repo, image, 10000000)
+    model.tag.create_temporary_hidden_tag(repo, image, 10000000) is not None
```

Comment:
I haven't ran this but should this be an assert statement?

Reply:
yes! good catch!